src/docs/product/integrations/integration-platform/ui-components/stacktrace-link.mdx

```diff
@@ -0,0 +1,43 @@
+---
+title: Stack Trace Link
+sidebar_order: 2
+description: "Learn about the stack trace link UI component."
+---
+
+## Stack Trace Link
+
+This feature allows the developer to insert a link within a stack trace frame. The link contains details about the project, file, line number, etc. It can also be used to view the file on another website, continue debugging in a different monitoring tool, etc. For example, this is what our Rookout integration looks like using stack trace linking.
```

Comment:
I'd change it to:

> This feature allows the developer to insert a link within a stack trace frame. The link contains details about the project, file name and line number. This can be used to view the file on your service or continue the debugging with a helpful reference.

No need to reference Rookout directly, it's in the screenshot